packages/2d/src/utils/patchSignal.ts

```diff
@@ -27,5 +27,12 @@ export function patchSignal<TSetterValue, TValue extends TSetterValue>(
     if (tweener) {
       signal.doTween = tweener.bind(owner);
     }
+    const computer = owner?.[`compute${capitalize(name)}`];
```

Comment:
Maybe it should be called `initial[Property]` or `default[Property]` or `getDefault[Property]`?
`compute` may be a bit confusing as it is only used when the signal is not overridden by the user.

Reply:
I'll leave that up to you; I don't really have a good idea for what to call it